<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyuanDialog</class>
 <widget class="QDialog" name="KeyuanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客源信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/images/tianjiakehu.png</normaloff>:/images/tianjiakehu.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>客户编号</string>
        </property>
        <property name="buddy">
         <cstring>kehuBianhaoLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="kehuBianhaoLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>客户需求</string>
        </property>
        <property name="buddy">
         <cstring>kehuXuqiuComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="kehuXuqiuComboBox">
        <item>
         <property name="text">
          <string>求购</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/images/tianjiakehu.png</normaloff>:/images/tianjiakehu.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>求租</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/images/tianjiakehu.png</normaloff>:/images/tianjiakehu.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>求合租</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/images/tianjiakehu.png</normaloff>:/images/tianjiakehu.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>当前状态</string>
        </property>
        <property name="buddy">
         <cstring>dangqianZhuangtaiComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="dangqianZhuangtaiComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>置业顾问</string>
        </property>
        <property name="buddy">
         <cstring>zhiyeGuwenComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="zhiyeGuwenComboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>客户来源</string>
        </property>
        <property name="buddy">
         <cstring>kehuLaiyuanComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="kehuLaiyuanComboBox"/>
      </item>
      <item row="1" column="4" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>138</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>求房情况</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>所在城区</string>
        </property>
        <property name="buddy">
         <cstring>suozaiChengquComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="suozaiChengquComboBox"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>房屋价格</string>
        </property>
        <property name="buddy">
         <cstring>fangwuJiageLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="fangwuJiageLineEdit"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>元</string>
        </property>
        <property name="buddy">
         <cstring>fangwuJiageLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>房屋类型</string>
        </property>
        <property name="buddy">
         <cstring>fangwuLeiXingComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fangwuLeiXingComboBox">
        <item>
         <property name="text">
          <string>1室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1室1厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2室1厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2室2厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3室1厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3室2厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>写字楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>办公室</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>商住两用</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>门面房</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>商铺</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>厂房</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>仓库</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>别墅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>地皮</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="chanquanXingBie">
        <property name="text">
         <string>产权|性别</string>
        </property>
        <property name="buddy">
         <cstring>chanquanXingbieComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="chanquanXingbieComboBox"/>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>房屋面积</string>
        </property>
        <property name="buddy">
         <cstring>fangwuMianjiDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="fangwuMianjiDoubleSpinBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>平方米</string>
        </property>
        <property name="buddy">
         <cstring>fangwuMianjiDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>新旧程度</string>
        </property>
        <property name="buddy">
         <cstring>xinjiuChengduComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="xinjiuChengduComboBox">
        <item>
         <property name="text">
          <string>全新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9成新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8成新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7成新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6成新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5成以下</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="5">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>所在楼层</string>
        </property>
        <property name="buddy">
         <cstring>suozaiLoucengSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="suozaiLoucengSpinBox"/>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>装修标准</string>
        </property>
        <property name="buddy">
         <cstring>zhuangxiuBiaozhunComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="zhuangxiuBiaozhunComboBox">
        <item>
         <property name="text">
          <string>毛坯房</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>简单装修</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中档装修</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>豪华装修</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="5">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>楼层总数</string>
        </property>
        <property name="buddy">
         <cstring>loucengZongshuSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="loucengZongshuSpinBox"/>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>意向地址</string>
        </property>
        <property name="buddy">
         <cstring>yixiangDizhiLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QLineEdit" name="yixiangDizhiLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>配套设施</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chufangCheckBox">
        <property name="text">
         <string>厨房</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chuangCheckBox">
        <property name="text">
         <string>床</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="jiajuCheckBox">
        <property name="text">
         <string>家具</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="youxianDianshiCheckBox">
        <property name="text">
         <string>有线电视</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="kuaidaiwangCheckBox">
        <property name="text">
         <string>宽带网</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="dianhuaCheckBox">
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="reshuiqiCheckBox">
        <property name="text">
         <string>热水器</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="dianshijiCheckBox">
        <property name="text">
         <string>电视机</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="kongtiaoCheckBox">
        <property name="text">
         <string>空调</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="xiyijiCheckBox">
        <property name="text">
         <string>洗衣机</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="bingxiangCheckBox">
        <property name="text">
         <string>冰箱</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="meiqiCheckBox">
        <property name="text">
         <string>煤气</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="nuanqiCheckBox">
        <property name="text">
         <string>暖气</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="qichekuCheckBox">
        <property name="text">
         <string>汽车库</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>房源详细信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="fangyuanXiangxiXinxiPlainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>保密信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>业主姓名</string>
        </property>
        <property name="buddy">
         <cstring>yezhuXingmingLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="yezhuXingmingLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>联系电话</string>
        </property>
        <property name="buddy">
         <cstring>lianxiDianhuaLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lianxiDianhuaLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>联系手机</string>
        </property>
        <property name="buddy">
         <cstring>lianxiShoujiLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lianxiShoujiLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>业主说明</string>
        </property>
        <property name="buddy">
         <cstring>yezhuShuomingLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="yezhuShuomingLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>271</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeyuanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyuanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
